omnibus/config/projects/supermarket.rb

```diff
@@ -30,8 +30,7 @@
 
 # NOTE: see the omnibus-supermarket cookbook gemfile for controlling the infra client version
 override :chef, version: "v17.5.22" #with the new change in chef we have to specify the exact version
-override :postgresql, version: '9.3.25'
-override :ruby, version: "2.7.5"
+override :ruby, version: "2.7.4"
```

Comment:
This needs to be 2.7.5 otherwise we're adding back CVEs

Reply:
sure

Reply:
Have updated.